src/helpers/normalize.js

```diff
@@ -52,6 +53,9 @@ const prepareKeys = entry => {
     }
   })
 
+  // Find and convert Firebase Timestamp objects to UTC strings
+  newEntry = parseFirebaseTimestamps(newEntry)
```

Comment:
@jperasmus I tested this with my sample Flamelink project and it caught all date fields correctly. However I'm not sure how to best test this for all data types / fields. Do you perhaps have a project with a lot of data and also one with nested entries (especially arrays) that we could test this against?

Reply:
Not sure if I need to test for RTDB. At this stage I'm not sure if there will be a difference in the data.

Reply:
The RTDB and CF treat dates differently. Only CF has these timestamps, but I don't see that it would break anything running this code on either.

I have this schema with literally all the Flamelink data fields that you can import and add some content. Just rename it to `.json` before you try and upload. Github doesn't allow upload .json files here.

[flamelink_all_the_things_schema.txt](https://github.com/flamelink/gatsby-source-flamelink/files/4628851/flamelink_all_the_things_schema.txt)

Reply:
Schweet, dankie. I'll test with this.